src/views/Messages.vue

```diff
@@ -89,27 +83,33 @@ export default {
       this.messages = messages // display all for now
       this.total_msg = response.data.pagination_total
     },
-    async getChannels() {
+    async getChannels () {
       let response = await axios.get(`${this.api_server}/api/v0/channels/list.json`)
       let channels = response.data.channels
 
       this.channels = channels // display all for now
     },
-    select_channel(channel) {
+    select_channel (channel) {
       this.selchannel = channel
       this.getMessages()
-    }
+    },
   },
   watch: {
-    async $route(to, from) {
+    async $route (to, from) {
       await this.refresh()
     },
-    async page() {
+    async page () {
       await this.refresh()
     }
   },
-  async created() {
+  async created () {
     await this.refresh()
   }
 }
 </script>
+
+<style>
+:root{
+  --vs-font-size: unset;
```

Comment:
What is this for ?

Reply:
The component I use (vue-select), uses [css variables to style the component](https://vue-select.org/guide/css.html#dark-mode-example), `--vs-font-size` is set to `1rem` by default which looks off with rest of the UI. 

Unsetting it, means that we use the default value provided by the parent.

Reply:
Would it make sense to add this explanation as a comment in this code ? I doubt we would remember later...

Reply:
Done